src/model.jl

```diff
@@ -991,3 +991,188 @@ function set_upper_bound(sector::SectorRef, u_bound::Float64)
 
     # s.model._sectors[s.index].upper_bound = u_bound
 end
+
+
+
+#Parameter::ScalarParameter, ::IndexedParameter
+
+#Commodity::ScalarCommodity, ::IndexedCommodity
+#Sector::ScalarSector,       ::IndexedSector
+#Consumer::ScalarConsumer,   ::IndexedConsumer
+#Aux::ScalarAux,             ::IndexedAux
```

Comment:
This can be removed, right?

Reply:
Yeah, absolutely. Reference for myself to ensure I hit all the points.